examples/specific_file.rs

```diff
@@ -14,5 +14,12 @@ fn main() {
 
     if let Ok(entry) = DesktopEntry::decode_from_path(path.to_path_buf(), locales) {
         println!("{}\n---\n{}", path.display(), entry);
+
+        dbg!(entry.comment(locales));
+
+        dbg!(entry.actions());
+        
+
+        dbg!(entry.action_entry("new-window", "Name"));
```

Comment:
Need to remove these dbg messages.

Reply:
It is more a testing purpose file, but done